### DISCUSSION TITLE
LG monitor OSD flicker on keypress

### DISCUSSION THREAD

**Discussion by razvanphp** [MonitorControl/MonitorControl#1129] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I'm using 38UC99 LG monitor and everything seems to be working, it's just a small annoyance that the monitor native OSD for volume flickers on each step/change of value, basically disappears and appears very quickly.

When changing the volume from the monitor button the OSD just stays on the screen and the dot moves.

### Steps to reproduce

Not sure if specific for my monitor, but just changing the volume by pressing a key, the OSD flickers.

### Expected behavior

The OSD remains on the screen while slider moves.

### Anything else?

I can maybe make a video if necessary...

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5
- Mac model: Macbook Pro 16, 2019
- MonitorControl version: 4.1.0 build 7034
- Monitor(s): LG 38UC99-W
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
This is an issue that comes up from time to time:

#711 #233 #157 #234 #462

There is no proper way to hide the OSD.

**Comment by razvanphp** [comment]
hello, thank you for the quick answer.

but.. that's not really the same thing, I actually disabled the macos OSD and want to keep the LG one, but it behaves differently when using the physical button and from this app, here's a small video:

https://user-images.githubusercontent.com/4599319/181587438-f5720351-f31a-48e9-851a-71f0581a19dc.mp4

**Reply by waydabber** [reply]
Oh, yes, I see. My LG did not do this blinking, but I guess this is a firmware version specific thing (my LG was a one generation older model - fun fact: unfortunately I just broke its screen during moving to a new place - got hit by the car's door frame as I was trying to rather vigorously close it despite the fact that the car was overstuffed with things :)).

**Reply by razvanphp** [reply]
oh, sorry to hear that. I can try some debug versions if you want to try to fix it blindly.

I just upgraded the firmware of the monitor last week from 3.07 to 3.11
